rest_introduction_app/api/challenges/challenge_6/challenge_6.py

```diff
@@ -88,32 +89,32 @@ async def add_to_backpack(body: Item):
 async def add_to_pocket(body: Item):
     if not is_pocket_size(body):
         raise HTTPException(status_code=403,
-                            detail=f"Are you trying to put {body.name} into your pocket? Really...")
+                            detail=f"Are you trying to put {body.name.name} into your pocket? Really...")
     if is_full(hiker.pocket):
         raise HTTPException(status_code=400,
                             detail=f"Your pocket is full already.")
     add_item(hiker.pocket, body.name)
     return JSONResponse({
-        "message": f"You've packed a {body.name}. {ITEM_RESPONSES.get(body.name)}."
+        "message": f"You've packed a {body.name.name}. {ITEM_RESPONSES.get(body.name.name)}"
     })
 
 
 @router.patch("/swap_backpack_item", status_code=status.HTTP_201_CREATED)
```

Comment:
@TomaszSzyborski, I don't know why this endpoint returns status code 200 instead of 201. Any idea?

Reply:
instead of JSONResponse return Object (dict)